airflow/migrations/versions/4addfa1236f1_add_fractional_seconds_to_mysql_tables.py

```diff
@@ -35,7 +35,7 @@
 
 
 def upgrade():  # noqa: D103
-    conn = op.get_bind()
+    conn = op.get_bind()  # pylint: disable=no-member
```

Comment:
I'm not sure this is strictly necessary - I copied it from other examples.

Do we not pylint migrations maybe?

Reply:
Yeah . Another consistency change maybe ?